test/sqllogictest/transform/fold_vs_dataflow/3_number_aggfns_dataflow.slt

```diff
@@ -37,7 +37,7 @@ SELECT
  AVG(real1 + real1), AVG(double1 + double1), AVG(numeric1 + numeric1)
 FROM t_using_dataflow_rendering;
 ----
--9223372036854775808  -9223372036854775808  100000000000000000000000000000000000002  -9223372036854775808  -9223372036854775808  200000000000000000000000000000000000004  2  2  2  4  4  4  9223372036854775807  9223372036854775807  100000000000000000000000000000000000000  9223372036854775807  9223372036854775807  200000000000000000000000000000000000000  -9223372036854775808  -9223372036854775808  9223372036854775807  -9223372036854775808  -9223372036854775808  9223372036854775807
+-9223372036854775808  -9223372036854775808  100000000000000000000000000000000000002  -9223372036854775808  -9223372036854775808  200000000000000000000000000000000000004  2  2  2  4  4  4  9223372036854775807  9223372036854775807  100000000000000000000000000000000000000  9223372036854775807  9223372036854775807  200000000000000000000000000000000000000  -9223372036854775808  -9223372036854775808  50000000000000000000000000000000000001  -9223372036854775808  -9223372036854775808  100000000000000000000000000000000000002
```

Comment:
I guess this is somehow due to the plan change around the numeric casts, but I haven't looked further into it.

Reply:
The changes are matching the folding result changes, right? If yes, then it's probably ok.